src/main/java/org/openstreetmap/atlas/checks/validation/linear/edges/RoundaboutMissingTagCheck.java

```diff
@@ -60,6 +67,10 @@ public RoundaboutMissingTagCheck(final Configuration configuration)
                 "angle.threshold.maximum_degree", MAX_THRESHOLD_DEGREES_DEFAULT, Angle::degrees);
         this.minAngleThreshold = this.configurationValue(configuration,
                 "angle.threshold.minimum_degree", MIN_THRESHOLD_DEGREES_DEFAULT, Angle::degrees);
+        this.tagFilterIgnore = this.configurationValue(configuration, "tags.filter",
```

Comment:
This configuration key is already defined in BaseCheck. While there is no loss of effect because BaseCheck is using it effectively the same way, it would be better to use a different key.

Reply:
thanks for suggestion! changed.